var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -30,14 +32,18 @@ class Googletest(CMakePackage):
     depends_on("c", type="build")
     depends_on("cxx", type="build")
 
+    variant("absl", default=False, when="@1.12.1:", description="Build with abseil and RE2")
+    depends_on("abseil-cpp", when="+absl")
+    depends_on("re2", when="+absl")
+
     variant("gmock", default=True, when="@1.8:", description="Build with gmock")
     variant("pthreads", default=True, description="Build multithreaded version with pthreads")
     variant("shared", default=True, description="Build shared libraries (DLLs)")
 
     variant(
         "cxxstd",
-        default="11",
-        values=("98", "11", "14", "17"),
+        default="14",
+        values=("98", "11", "14", "17", "20"),
```

Comment:
From when is cxxstd=20 supported? Until when is cxxstd below the default supported?

Reply:
c++14 is required from 1.13 and above (there's a `conflicts("cxxstd=11", when="@1.13:")`) and it's unknown if or when the previous builds were incompatible with C++20. I figure if anyone tries to install an older version with a newer standard and encounters a build error, we can add a conflict then?